.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.7"
+cache: pip
+install:
+  - pip install -r requirements.txt
+  - pip install -r requirements-dev.txt
+script:
+  - make format-check
+  - pylint --output-format=text main.py
```

Comment:
why you lint `main.py` again? it has been already checked in `make format-check`.

Reply:
Thanks for reviewing it
Now I have removed it.
Is there any other change?

Reply:
Thanks! This looks good to me but you will need to ask the repo owner @srush to see if this is what he planned.